lib/Core/Executor.cpp

```diff
@@ -3966,6 +4052,13 @@ void Executor::runFunctionAsMain(Function *f,
 
   globalObjects.clear();
   globalAddresses.clear();
+  std::map<ExecutionState*, Assignment *>::iterator it = maxForksMap.begin();
```

Comment:
Use a C++11 iterator: `for (auto& [state, asignment]: maxForksMap)`

Reply:
The `[state, assigment]` is C++17

Reply:
I thought that's now fine now?  Or are we still at C++11?

Reply:
We are at C++14.